### PR TITLE
Cleaning up @since TODO / @since XXX

### PR DESCRIPTION
**Pull request by daniel-beck**
I've left out most LTS releases and only noted the weeklies, since new API shouldn't be particularly relevant for those (and are likely to be harmful).

@jenkinsci/code-reviewers

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
#2571 addresses Console Commands. I'd rather exclude them from this PR in order to avoid conflicts

**Comment by oleg-nenashev** [comment]
But thanks for doing it in any case. Was somewhere in my TODO list, but I've never addressed it completely

**Comment by oleg-nenashev** [comment]
PWNED by #2576. @batmat

**Comment by daniel-beck** [comment]
@oleg-nenashev Why would that be a close reason?

**Comment by oleg-nenashev** [comment]
@daniel-beck Sorry, just assigned wrong label

**Review by KostyaSha — APPROVED** [review]

**Comment by oleg-nenashev** [comment]
Still a merge conflict

**Comment by daniel-beck** [comment]
Fixed the merge conflicts, one line endings file, two where #2571 got it wrong.

File: core/src/main/java/hudson/util/ProcessKillingVeto.java [hunk 1]
```diff
@@ -100,4 +100,4 @@ public VetoCause(@Nonnull String message) {
      */
     @CheckForNull
     public abstract VetoCause vetoProcessKilling(@Nonnull IOSProcess p);
-}
+}
```
Comment (oleg-nenashev) [inline core/src/main/java/hudson/util/ProcessKillingVeto.java]: ORLY?

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
Line ending got replaced by a wrong one, but other changes LGTM

File: core/src/main/java/hudson/util/ProcessKillingVeto.java (hunk 1 above)
Comment (KostyaSha) [inline core/src/main/java/hudson/util/ProcessKillingVeto.java]: but who cares... other prs reformatting code also

**Comment by oleg-nenashev** [comment]
OK, I agree with @KostyaSha it is not that important

**Review by oleg-nenashev — APPROVED** [review]

**Comment by daniel-beck** [comment]
@oleg-nenashev Fixed the missing line ending in the last line.

Fixing merge conflicts when all line endings are changed is a PITA :frowning_face:

**Comment by oleg-nenashev** [comment]
re-:+1:

**Review by batmat — APPROVED** [review]
:+1: 
Yeah, sorry, was a necessary evil. The good news is it was a one-time to now make mixing newline formats impossible.

**Pull request merged**